packages/core/src/sass/globals.scss

```diff
@@ -47,7 +47,7 @@ $uikit-maxwidth: 42em; //Answer to life, the universe, and everything (keeps thi
 $uikit-colour-Hero: #115361 !global; //color variables (yes! the actual spelling doesn't have a u)
```

Comment:
👍  we had this underlying issue with Dashboards early on as well, caused so many magical issues.  Also funnily, IDP has this issue recently.